.gitignore

```
@@ -74,6 +74,7 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
 fastlane/promo_screenshots
+/fastlane/metadata/review_information
```

You probably didn't mean to have that leading `/` in here…

```suggestion
fastlane/metadata/review_information
```

Never seen this directory personally, what is it supposed to contain?

---

> You probably didn't mean to have that leading / in here…

True, fixed in 80f9b7dd6f11c660da64917ff3788c10feb148a8 because I didn't think to just hit the button in the dropdown 🤦

---

> Never seen this directory personally, what is it supposed to contain?

It contains the name and phone number of the App Store Connect contact, as well as the demo credentials used for evaluating the app. 

We don't want those to be public, which is why they're excluded.